bika/lims/browser/worksheet/views/analyses.py

```diff
@@ -384,11 +389,25 @@ def get_slot_header(self, item):
         elif obj.portal_type == 'ReferenceAnalysis' and obj.ReferenceType == 'c':
             pos_text += "<a href='%s'><img title='%s' src='++resource++bika.lims.images/control.png'></a>" % (
             parent.absolute_url(), parent.Title())
+            api_url = self.portal_url + \
+                      "/@@API/read?catalog_name=uid_catalog" \
+                      "&include_fields=Remarks&UID=%s"%(parent.UID())
```

Comment:
The JSON API URL should be never directly rendered into the HTML, even if you put an JS event handler afterwards on it. A better approach is to do that in the `click` handler.

Reply:
Done, moved code from "click" to overlay handler.